data/transactions/verify/txn.go

```diff
@@ -346,12 +353,21 @@ func logicSigBatchVerify(txn *transactions.SignedTxn, groupIndex int, groupCtx *
 	if groupIndex < 0 {
 		return errors.New("Negative groupIndex")
 	}
+
+	var ts int64
+	if groupCtx.consensusParams.EnableFirstValidTimeStamp {
```

Comment:
I'm not sure if this test is needed here, since the teal 5 won't be enabled anyway.

Reply:
True, but I'm worried about any difference in behavior caused by calling `GetBlockTimeStamp` and erroring for whatever reason (retrieving of the timestamp happens for all programs)